### ISSUE TITLE
The alpha of the watermark is not preserved if you place the watermark and apply opacity when using the Imagick driver

### ISSUE THREAD

**Issue by ludekbenedik** [Intervention/image#1367]
**Describe the bug**
![result-imagick](https://github.com/Intervention/image/assets/454618/8f31c482-561a-4968-a4b9-7d7fbc51e7c8)

**Code Example**
```php
$manager = ImageManager::imagick();
$image = $manager->read('image.png');
$image->place('watermark.png', 'center-center', opacity: 50);
$image->save('result-imagick.png');
```

**Expected behavior**
(result when using the GD driver)
![result-gd](https://github.com/Intervention/image/assets/454618/18f24d64-dbf1-4f9a-8748-b91ad8b484e6)

**Images**
Image
![image](https://github.com/Intervention/image/assets/454618/6aba1baa-30d9-486c-bdfa-b2ffb7764a58)

Watermark
![watermark](https://github.com/Intervention/image/assets/454618/7b8297e5-1761-44e9-95ca-eeee7beeadcf)

**Environment (please complete the following information):**
 - PHP Version: 8.1.29
 - OS: Ubuntu 22.04 (kernel 6.5.0)
 - Intervention Image Version: 3.6.5
 - GD or Imagick: Imagick
 - imagick module version: 3.7.0
 -  ImageMagick library version: 6.9.11-60 Q16 x86_64 2021-01-25

**Comment by olivervogel** [comment]
Thanks for reporting. I will look into it.

**Comment by olivervogel** [comment]
This issue has been fixed as of version [3.7.0](https://github.com/Intervention/image/releases/tag/3.7.0).
